central/policycategory/datastore/datastore_impl.go

```diff
@@ -35,8 +32,7 @@ var (
 			sac.AccessModeScopeKeys(storage.Access_READ_ACCESS, storage.Access_READ_WRITE_ACCESS),
 			sac.ResourceScopeKeys(resources.WorkflowAdministration)))
 
-	titleCase          = cases.Title(language.English, cases.NoLower)
-	incorrectTitleCase = cases.Title(language.English)
+	titleCase = cases.Title(language.English, cases.NoLower)
```

Comment:
@clickboo I think this is the line you were looking for.  It is still there.  The revert just undoes the migration-like stuff.